help-woke.rst

```diff
@@ -0,0 +1,46 @@
+:orphan:
+
+===================================
+Inclusive language check exemptions
+===================================
+
+This page provides an overview of two inclusive language check exemption
+methods for files written in reST format. See the `woke documentation`_ for
+full coverage.
+
+Exempt a word
+-------------
+
+To exempt an individual word, place a comment on a line immediately preceding
+the line containing the word in question. This special comment is to include
+the syntax ``wokeignore:rule=<SOME_WORD>``. For instance:
+
+.. code-block:: none
+
+   .. wokeignore:rule=whitelist
+   This is your text. The word in question is here: whitelist. More text.
+
+Here is an example of an exemption that acts upon an element of a URL that is
+expressed using the link definition method (typically at the bottom of a file):
```

Comment:
I don't find that clearer - will everyone who uses the starter pack know what the "link definition method" is?

The issue that we see a lot is that a URL contains "master", and that this is hard to fix with a comment in the line above (especially in MyST, btw). So what we should say is that for links, the best solution is to define them in a separate file or at the bottom of the page and put the comment in the line above the link definition.

Reply:
I will propose something.